src/mlpack/bindings/python/print_class_defn.hpp

```diff
@@ -59,20 +59,20 @@ void PrintClassDefn(
   /**
    * This will produce code like:
    *
-   * cdef class <ModelType>Type:
-   *   cdef <ModelType>* modelptr
+   * cdef class <ModelType\>Type:
```

Comment:
Maybe we should put this in an `@code` block instead?  Then we wouldn't need the various `\` escape characters.